lib/plugin/visitors/model-class.visitor.ts

```diff
@@ -52,23 +51,33 @@ export class ModelClassVisitor extends AbstractFileVisitor {
             typeChecker,
             options,
             sourceFile.fileName,
-            sourceFile
+            sourceFile,
+            metadata
           );
         } catch (err) {
           return node;
         }
         return node;
       }
-      return ts.visitEachChild(node, visitNode, ctx);
+      return ts.visitEachChild(node, visitClassNode, ctx);
```

Comment:
I’m sorry, I just realized its incorrect to pass visitClassNode here. I will fix PR today